server/services/prisonService.ts

```diff
@@ -38,6 +38,20 @@ export default class PrisonService {
     }
   }
 
+  /**
```

Comment:
I like this implementation and the unit tests above 👍 

Of interest is the fact that the service method does not throw/return an error. It catches any error and returns an empty map 👍 
Thats perfect, but it means the comments I made above re: the educationAndWorkPlanService and whether prisonService can/should throw errors and set an error response probably have some merit?
What do you think? 🤔

Reply:
ha sorry I should've explained this probably. As you mentioned, in reality it never returns a rejected promise and instead returns an empty list so you will just get the prison ID instead (made an executive decision on that one). 

I felt like I should test what happens if the promise is rejected and built up tests before I got to the actual implementation and the service should probably not be aware of the implementation details (maybe there is some javascripty way it can reject, e.g., NPE?). 

I'll change it to try-catch and default to empty list rather than returning a problem retrieving data.

Reply:
Updated to catch and return an empty map